vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc

```diff
@@ -475,7 +475,7 @@ void Contribution::AddRetry(
     braveledger_bat_helper::CURRENT_RECONCILE reconcile) {
   BLOG(ledger_, ledger::LogLevel::LOG_WARNING)
       << "Re-trying contribution for step"
-      << std::to_string(step)
+      << std::to_string(static_cast<int32_t>(step))
```

Comment:
is cast needed here?

Reply:
Yes. Since the enum is now in mojo it is an enum class so you need the cast.